src/components/Login/Login.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import { Link, useNavigate } from 'react-router-dom';
+
+interface ILogin {
+	email: string;
+	password: string;
+}
+
+function Login() {
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+	const [hasError, setError] = useState(false);
+	const navigate = useNavigate();
+
+	const sendRequest = async () => {
+		try {
+			const response = await fetch('http://localhost:4000/login', {
+				method: 'POST',
+				body: JSON.stringify({ email, password }),
+				headers: {
+					'Content-Type': 'application/json',
+				},
+			});
+			if (!response.ok) {
+				throw new Error('Something went wrong');
```

Comment:
Try to come up with a more descriptive error message.
When you see such an error in the console it is probably better if it says that there is something wrong with the response from the API, not that just something went wrong don't you think?